Windows/Debugger/CtrlMemView.cpp

```diff
@@ -634,13 +631,49 @@ void CtrlMemView::scrollCursor(int bytes)
 	redraw();
 }
 
+
+std::vector<u32> CtrlMemView::searchString(std::string searchQuery)
+{
+	
+	std::vector<u32> searchResAddrs;
+	std::vector<u8> searchData;
+
+	auto memLock = Memory::Lock();
+	if (!PSP_IsInited())
+		return searchResAddrs;
+	
+	int queryLength = size_t(searchQuery.length());
```

Comment:
nit: `size_t(searchQuery.length())` -> `(int)searchQuery.length()` to prevent a warning, I'd assume?

-[Unknown]

Reply:
I didn't want to do a cast within the memcmp, since that needs a size_t